.github/workflows/main.yaml

```diff
@@ -0,0 +1,99 @@
+name: main
```

Comment:
Could you change the name to "Singleton-Angular-Client", then we can distinguish it in Actions well?

Reply:
Updated